how-to-guides/data-versioning/notebooks/Organize_and_share_dataset_versions.ipynb

```diff
@@ -193,34 +183,9 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "{'datasets': {'train_sampled': {'latest': <neptune.new.attributes.atoms.artifact.Artifact at 0x7fa0ddfa2650>,\n",
```

Comment:
maybe this structure can be copy&pasted as markdown cell -> it really helps understand this structure.